template/{{cookiecutter.repository_name}}/operations/evaluate.py

```diff
@@ -99,7 +101,6 @@ def evaluate_batch(engine, examples):
         run_id=os.getenv('RUN_ID'),
     )
 
-    json.write(config, 'config.json')
+    Path('config.json').write_text(json.dumps(config))
```

Comment:
A bit Rust damaged probably, but you have a temporary string just to write it down to disc ;)

Reply:
I think there may be a misunderstanding. We have removed the `json` module from workflow so we can't do `json.write` anymore.

Reply:
but json.dumps works?

Reply:
ah, right  the wrapper?

Reply:
`json` is a standard library

Reply:
```python
with open('config.json') as fp:
    json.dump(config, fp)
```

Reply:
Not saying its pretty ;P

Reply:
keep it as is I think

Reply:
That is the old version and the reason why we created `json.write` and `json.read` because it was so ugly